analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Utilities/SymbolReferenceAnalyzer/Property.vb

```diff
@@ -0,0 +1,10 @@
+﻿Public Class Sample
+
+    Public Property Name As String
+
+    Public Sub Method()
+        Dim name As String = Name
+        Name = "John"
```

Comment:
This doesn't do what you expect at all 😆 
First, it self-assigned the fresly declared variable `name` to itself.
Then you assign `"John"` back to this local variable.

It can be fixed like this:
```suggestion
        Dim n As String = Name
        Name = "John"
```